### PR TITLE
LPS-126216, LPS-126222 | master

### PR DESCRIPTION
**Pull request by david-gutierrez-mesa**
https://issues.liferay.com/browse/LPS-126216
https://issues.liferay.com/browse/LPS-126222
cc/ @nikki-pru

### CODE REVIEW

**Comment by david-gutierrez-mesa** [comment]
No related failures. See https://github.com/david-gutierrez-mesa/liferay-portal/pull/117#issuecomment-773439547

<html><h3>:heavy_check_mark: ci:test:stable - 9 out of 9 jobs passed</h3><h3>:x: ci:test:relevant - 22 out of 26 jobs passed in 4 hours 9 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/4f5dd4c5c07162fa9e76f543b6c908069733a123">4f5dd4c5c07162fa9e76f543b6c908069733a123</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/4f5dd4c5c07162fa9e76f543b6c908069733a123">4f5dd4c5c07162fa9e76f543b6c908069733a123</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//1513">Acceptance Upstream DXP (master) #1513</a></p><details><summary><strong>ci:test:stable - 9 out of 9 jobs PASSED</strong></summary><details><summary><strong>9 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/451870/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463663/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463540/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/451872/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/451873/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463541/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403443/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463537/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463662/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 22 out of 26 jobs PASSED</strong></summary><div><h4>4 Failed Jobs:</h4><ul><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest(master)/7320/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403445/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/451874/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</strong></strike></a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/433214/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>22 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/424221/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463539/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/451870/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/348179/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/424223/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/318095/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463663/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463540/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/461396/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/461384/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463538/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/451872/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403444/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463661/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/451873/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/451871/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463541/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403443/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/424222/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463537/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463536/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/463662/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-23.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7320/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403445/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>93 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=8,label_exp=!master/403445//consoleText">AXIS_VARIABLE=8,label_exp=!master #403445</a><ol><li><div><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=8,label_exp=!master/403445//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_AppBuilder_UpdateDeployment_">LocalFile.AppBuilder#UpdateDeployment</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-23/1612441396907/test-portal-acceptance-pullrequest(master)/7320/functional-tomcat90-mysql57-jdk8/0/8/LocalFile.AppBuilder_UpdateDeployment/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-23/1612441396907/test-portal-acceptance-pullrequest(master)/7320/functional-tomcat90-mysql57-jdk8/0/8/LocalFile.AppBuilder_UpdateDeployment/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-23/1612441396907/test-portal-acceptance-pullrequest(master)/7320/functional-tomcat90-mysql57-jdk8/0/8/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/451874/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></h5><div><h6>Job Results:</h6><p>65 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=4,label_exp=!master/451874//consoleText">AXIS_VARIABLE=4,label_exp=!master #451874</a><ol><li><div><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=4,label_exp=!master/451874//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_AppBuilderWorkflow_ValidateFormViewModes_">LocalFile.AppBuilderWorkflow#ValidateFormViewModes</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-23/1612441396907/test-portal-acceptance-pullrequest(master)/7320/functional-tomcat90-mysql57-jdk8/1/4/LocalFile.AppBuilderWorkflow_ValidateFormViewModes/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-23/1612441396907/test-portal-acceptance-pullrequest(master)/7320/functional-tomcat90-mysql57-jdk8/1/4/LocalFile.AppBuilderWorkflow_ValidateFormViewModes/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-23/1612441396907/test-portal-acceptance-pullrequest(master)/7320/functional-tomcat90-mysql57-jdk8/1/4/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 4f5dd4c5c07162fa9e76f543b6c908069733a123:</strong></summary><ol><li><div><h5><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/451874/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></h5><div><h6>Job Results:</h6><p>65 Tests Passed.<br/>2 Tests Failed.</p></div><ol><li><div><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=23,label_exp=!master/451874//consoleText">AXIS_VARIABLE=23,label_exp=!master #451874</a><ol><li><div><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=23,label_exp=!master/451874//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_CPDynamicdatalists_AddListRecordWithLinkToPageField_">LocalFile.CPDynamicdatalists#AddListRecordWithLinkToPageField</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-23/1612441396907/test-portal-acceptance-pullrequest(master)/7320/functional-tomcat90-mysql57-jdk8/1/23/LocalFile.CPDynamicdatalists_AddListRecordWithLinkToPageField/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-23/1612441396907/test-portal-acceptance-pullrequest(master)/7320/functional-tomcat90-mysql57-jdk8/1/23/LocalFile.CPDynamicdatalists_AddListRecordWithLinkToPageField/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-23/1612441396907/test-portal-acceptance-pullrequest(master)/7320/functional-tomcat90-mysql57-jdk8/1/23/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=23,label_exp=!master/451874//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_CPDynamicdatalists_AddFormTemplate_">LocalFile.CPDynamicdatalists#AddFormTemplate</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-23/1612441396907/test-portal-acceptance-pullrequest(master)/7320/functional-tomcat90-mysql57-jdk8/1/23/LocalFile.CPDynamicdatalists_AddFormTemplate/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-23/1612441396907/test-portal-acceptance-pullrequest(master)/7320/functional-tomcat90-mysql57-jdk8/1/23/LocalFile.CPDynamicdatalists_AddFormTemplate/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-23/1612441396907/test-portal-acceptance-pullrequest(master)/7320/functional-tomcat90-mysql57-jdk8/1/23/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/433214/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>495 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/433214//consoleText">AXIS_VARIABLE=1,label_exp=!master #433214</a><ol><li><div><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/433214//testReport/com.liferay.data.engine.rest.resource.v2_0.test/DataDefinitionResourceTest/testPostDataDefinitionByContentType">DataDefinitionResourceTest.testPostDataDefinitionByContentType</a><pre><code>java.lang.AssertionError: An exception must be thrown
	at org.junit.Assert.fail(Assert.java:89)
	at com.liferay.data.engine.rest.resource.v2_0.test.DataDefinitionResourceTest.testPostDataDefinitionByContentType(DataDefinitionResourceTest.java:428)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$3.evaluate(TestExecutorRunnable.java:353)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at org.junit.rules....</code></pre></div></li></ol></div></li></ol></div></li></ol></details></details></html>

**Comment by david-gutierrez-mesa** [comment]
ci:reopen

**Comment by Tim-Cao** [comment]
ci:close

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase [hunk 1]
```diff
@@ -123,12 +160,21 @@ definition {
 	}
 
 	@description = "This is a test for LPS-123539. As Content Creator, I want to leverage the structures’ translations with fildset when creating a new Web Content"
-	@ignore = "true"
 	@priority = "4"
 	test DefaultTitlePersistsUponFirstlyTranslatingForAnotherLocale {
-		property portal.acceptance = "false";
+		WebContentNavigator.gotoAddCP();
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase:165]: Should add fieldset not add web content based on Basic Web Content

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase [hunk 2]
```diff
@@ -123,12 +160,21 @@ definition {
...
+
+		PortletEntry.changeLocale(locale = "pt-BR");
+
+		PortletEntry.inputTitle(title = "BR Title");
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase:169]: Should add localized  value

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase [hunk 3]
```diff
@@ -123,12 +160,21 @@ definition {
...
+
+		PortletEntry.changeLocale(locale = "en-US");
+
+		PortletEntry.inputTitle(title = "EN Title");
+
+		PortletEntry.changeLocale(locale = "pt-BR");
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase:175]: Add step to publish web content then check

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase [hunk 4]
```diff
@@ -104,13 +106,48 @@ definition {
...
+		WebContentStructures.saveCP(structureName = "Structure Name Test");
+
+		NavItem.gotoWebContent();
+
+		WebContentNavigator.gotoAddWithStructureCP(structureName = "Structure Name Test");
 
-		// TODO LPS-123539 DefaultTitleIsDisplayedUponLocaleChange pending implementation
+		PortletEntry.inputTitle(title = "WC Title Test");
+
+		PortletEntry.changeLocale(locale = "pt_BR");
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase:140]: Add step to add localized title in one language

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase [hunk 5]
```diff
@@ -104,13 +106,48 @@ definition {
...
+		AssertTextEquals(
+			locator1 = "TextInput#TITLE",
+			value1 = "WC Title Test");
+
+		PortletEntry.changeLocale(locale = "es_ES");
+
+		AssertTextEquals(
+			locator1 = "TextInput#TITLE",
+			value1 = "WC Title Test");
 	}
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase:151]: Add step to publish web content then check default title is shown in default language and non translated language

**Comment by Tim-Cao** [comment]
Hi @david-gutierrez-mesa, One case steps are inconsistent with the goal of case.
The other one miss some validation steps that could miss bugs like https://issues.liferay.com/browse/LPS-125445
WebContent.viewWebContentTitle had better to be replaced by WebContent.viewCP
https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/macros/journal/WebContent.macro#L1760
Just add a if condition like if(isSet(webContentContent)) .
Since there is WebContent.viewTitle existing, while it validates the title in Web Content admin not in editor.
WebContent.viewWebContentTitle and viewTitle could confuse users.

**Comment by david-gutierrez-mesa** [comment]
Thank you @Tim-Cao I'm taking a look.

**Comment by nikki-pru** [comment]
Thank you @Tim-Cao and @david-gutierrez-mesa , I can send back to the team for fixes.

**Comment by david-gutierrez-mesa** [comment]
@nikki-pru don't worry! I already did the fixes :-). See https://github.com/david-gutierrez-mesa/liferay-portal/pull/118

**Pull request closed without merging**